benchmarks/index.md

```diff
@@ -84,7 +84,7 @@ codes, but were still designed to be not too computationally
 demanding. The National Institute of Standards and Technology has
 played a coordinating role in the development and management of these
 problems, and also hosts a
-[website for the Micromagnetic Standard Problems](http://www.ctcms.nist.gov/~rdm/mumag.org.html){:target="_blank"},
+[website for the Micromagnetic Standard Problems][mumag]{:target="_blank"},
```

Comment:
This link appears to be no longer working on the test site. Did you add the mumag link?

Reply:
I did, then removed it, then added again... Fixed.